app-template/yarn.lock

```diff
@@ -8609,6 +8677,12 @@ to-regex-range@^2.1.0:
     is-number "^3.0.0"
     repeat-string "^1.6.1"
 
+to-regex-range@^5.0.1:
```

Comment:
where did all of the new regex helper libraries come from?

Reply:
I ran `yarn why to-regex-range` and
got 
```
=> Found "to-regex-range@2.1.1"
info Reasons this module exists
   - "braces#fill-range" depends on it
   - Hoisted from "braces#fill-range#to-regex-range"
info Disk size without dependencies: "16MB"
info Disk size with unique dependencies: "48MB"
info Disk size with transitive dependencies: "88MB"
info Number of shared dependencies: 4
=> Found "sass#to-regex-range@5.0.1"
info Reasons this module exists
   - "sass#chokidar#braces#fill-range" depends on it
   - Hoisted from "sass#chokidar#braces#fill-range#to-regex-range"
info Disk size without dependencies: "16MB"
info Disk size with unique dependencies: "32MB"
info Disk size with transitive dependencies: "72MB"
info Number of shared dependencies: 3
```

it looks like sass got a [patch version bump](https://github.com/Sage-Bionetworks/portals/pull/41/files#diff-639272fd12ecc4bdbe3a16f135ce72abR7867) and these all downloaded as a result